### ISSUE TITLE
Cash Flow Visualization

### ISSUE THREAD

**Issue by smurphy8** [codeforokc/openbudgetokc#55]
Implement a visualization to show how cash goes from revenue to expense.  
Modeled after the cash flow visual in the open-budget-okc.
http://openbudgetoakland.org/2015-17-adopted-budget-flow.html

Notes for this are referenced in #36 . @smurphy

**Comment by thirdcreed** [comment]
Anyone on this?  I could make a working version in the sandbox, don't want to step on toes though.

**Comment by smurphy8** [comment]
I don't think so.

**Comment by smurphy8** [comment]
Also, the sandbox is just one way of doing things.  You can also do things
in place in a branch if you want.  It makes getting the code separated a
bit easier

**Comment by thirdcreed** [comment]
Understood, I might just make a codepen, (with the data hardcoded) and then put it directly in source on a branch.  There's nothing proprietary, about this data correct?  I mean, I can just put it in the codepen and not worry about people seeing it correct?

**Comment by dashcraft** [comment]
@thirdcreed @smurphy8 Did any work get done on this? I can take a look at it later if it's still open!

**Comment by jagthedrummer** [comment]
@thirdcreed You're correct that this data is not sensitive. Using codepen for it is perfectly fine.
